pkgs/desktops/gnome/extensions/no-title-bar/default.nix

```diff
@@ -23,7 +23,10 @@ stdenv.mkDerivation rec {
 
   makeFlags = [ "INSTALLBASE=$(out)/share/gnome-shell/extensions" ];
 
-  uuid = "no-title-bar@jonaspoehler.de";
+  passthru = {
+    extensionUuid = "no-title-bar@jonaspoehler.de";
+    extensionPortalSlug = "no-title-bar";
```

Comment:
Probably should comment that actual slug is `no-title-bar-forked`

Reply:
Thank you, good catch! No, I think I should set it to `no-title-bar-forked` and then add it to the renames list.

Reply:
Should we remove this here then?

Reply:
Not sure. Actually, I planned to add this to all extensions so that it's more consistent, but I didn't get to / was to lazy and there was no urgent need.

Reply:
If we add this to all extensions, it either should match the actual slug (mostly useless IMO and could confuse people expecting it to match the attribute name) or it should be the corrected version but then the change should be noted in a comment.

Reply:
But should we? (Also, I'm starting to think of simply renaming `extensionPortalSlug` to something like `attributeName` to make sure it should only be used for attribute name calculation.)

Reply:
I do not have a preference either way. The reason I went with `extensionPortalSlug` is precisely because it does not have to be attribute name in the case of automatically packaged extensions – it can be renamed.

Reply:
I see. Then I suggest keeping it as is for the sake of getting this merged eventually.

Reply:
The comments or removal would be nice for the cases where there is the discrepancy, like here, though.